docs/MPURegionsUsageGuide.md

```diff
@@ -30,41 +40,48 @@
 ```
 
 ## How to control MPU
-1. **How to add new MPU region**
-	- Logical mapping of MPU regions to their respective memory segments is done statically using below enumeration in common mpu.h header file (os/include/tinyara/mpu.h).
+1. **Board specific MPU region reservation**
+	- In order to support few regions for the board specific purpose, below API is provided to reserve the first *num* number of regions:
+	```c
+	void mpu_set_nregion_board_specific(uint8_t num);
+	```
+	Above function shall be called during board initialization after the board specific MPU regions have been configured. The TizenRT MPU regions are configured after *num* number of board MPU regions.
+
+2. **MPU usage overview**
+	- Logical mapping of MPU regions to their respective memory segments is done statically using below structure in common mpu.h header file (os/include/tinyara/mpu.h).
+	```c
+	struct mpu_usages_s {
+	        uint8_t nregion_board_specific;
+	        uint8_t nregion_common_bin;
+	        uint8_t nregion_app_bin;
+	        uint8_t nregion_stackovf;
+	        uint8_t max_nregion;
+	};
+	```
+	- This structure holds the region number range used for each type of mpu region (board specific, application binary, common binary, etc) and these are initialized during system boot using below API:
+	```c
+	void mpu_region_initialize(struct mpu_usages_s *mpu)
+
+	offset += MPU_NUM_REGIONS;
+	mpu->nregion_xxx_xx = offset;
+	```
+3. **How to add new MPU region**
```

Comment:
Can we follow below sequence?
1. MPU setting overview
2. Board specific MPU region reservation
3 How to set TizenRT MPU region to real MPU hardware
4. How to add new MPU region
...

Reply:
Done